src/content/public goods funding.md

```diff
@@ -0,0 +1,281 @@
+
+2. **Bribery and collusion**: It is possible to bribe voters to donate to a particular project (even if they prefer another option), thereby artificially inflating perceived preference for the former. This defeats the purpose of voting since the votes cast by a bribed individual may not reflect their true preference. Hence, just like Sybil attacks, bribery and collusion reduce the effectiveness of quadratic funding mechanisms. 
+
+Ethereum solves both problems in the following ways:
+
+### Decentralized identity {#decentralized-identity} 
+
+[Decentralized identity](/decentralized-identity/) allows individuals to verify their identity online without relying on a central authority, such as governments or service providers. This is usually done in a way that minimizes the exposure of personal information to third parties. 
+
+Some [decentralized identity protocols](/decentralized-identity/#use-decentralized-identity) have built-in cryptoeconomic incentives to discourage users from creating multiple identities to game the system. By integrating such tools with the voting process, quadratic funding applications reduce the risk of Sybil attacks. 
+
+### Zero-knowledge proofs {#zero-knowledge-proofs} 
+
+[Zero-knowledge proofs](/zero-knowledge-proofs/) enable anyone to prove the validity of a claim without exposing the underlying information. For example, Alice can prove her status as a United States citizen without having to provide her passport details. 
+
+Zero-knowledge proofs are particularly useful for preventing bribery and collusion in public voting. This is achieved by obfuscating details about individual votes whilst preserving the integrity of voting results. 
+
+Bribe-for-vote schemes only work if the bribee can prove they voted for the briber's preferred option. This is relatively easy to execute on an online voting platform, more so on Ethereum where all information is public. 
+
+But zero-knowledge proofs ensure individuals can prove they voted in an election without having to reveal how they voted. [Minimum Anti-Collusion Infrastructure](https://github.com/privacy-scaling-explorations/maci/blob/master/specs/01_introduction.md), which uses smart contracts, is an implementation of this technique and has been successfully used to protect quadratic funding from bribery and collusion. 
+
+Using MACI, it is possible to correctly tally results from a qudratic voting round and verify that an individual participated, while hiding details of how they voted. Since bribers cannot possibly know how a bribed individual, they have less incentive to hand out bribes in hopes of swinging votes.
```

Comment:
```suggestion
Using MACI, it is possible to correctly tally results from a quadratic voting round and verify that an individual participated, while hiding details of how they voted. Since bribers cannot possibly know how a bribed individual, they have less incentive to hand out bribes in hopes of swinging votes.
```